virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -40,3 +43,43 @@ def create_disk(disk_type, path=None, size="500M", disk_format="raw", extra='',
     else:
         return libvirt.create_local_disk(disk_type, path=path, size=size,
                                          disk_format=disk_format, extra=extra)
+
+
+def create_primitive_disk(type_name, disk_device, device_target, device_bus,
+                          device_format, disk_src_dict, disk_auth_dict):
+    """
+    Creates primitive disk xml
+
+    usage: disk_auth_dict = {"auth_user": ceph_auth_user,
+                              "secret_type": auth_sec_usage_type,
+                              "secret_uuid": auth_sec_uuid}
```

Comment:
@chunfuwen Please check if the usage is copy error here, thanks

Reply:
updated